solvers/minion/build.sh

```diff
@@ -2,20 +2,17 @@
 
 SCRIPT_DIR=$(dirname "$0")
 
-git submodule init
-git submodule update
+git submodule init -- vendor
+git submodule sync -- vendor 
+git submodule update --init --recursive -- vendor
+
 cd "$SCRIPT_DIR"
```

Comment:
it looks like this cd wants to be before the git commands, am I missing something?

Reply:
thank you - thats a typo on my part.